ci-operator/config/openshift/api/openshift-api-master.yaml

```diff
@@ -88,9 +88,13 @@ tests:
     dir=$GOPATH/src/github.com/openshift/client-go
     git clone https://github.com/openshift/client-go --branch ${PULL_BASE_REF:-master} --single-branch $dir
     rm -rf $dir/vendor/github.com/openshift/api/*
-    cp -Rf * $dir/vendor/github.com/openshift/api/
-    rm -rf $dir/vendor/github.com/openshift/api/vendor
+    cp -Rf * $dir/api/
```

Comment:
There is not dir for `/src/github.com/openshift/client-go/api` so you may need to create one or do a temp one.

Reply:
Or I can force to copy the entire current directory `./` rather than its contents `*` :wink: